src/h5web/Breadcrumbs.tsx

```diff
@@ -0,0 +1,68 @@
+import { Fragment, ReactElement, useContext } from 'react';
+import { FiChevronRight } from 'react-icons/fi';
+import styles from './BreadcrumbsBar.module.css';
+import { assertAbsolutePath } from './guards';
+import { ProviderContext } from './providers/context';
+
+interface Props {
+  path: string;
+  onSelect: (path: string) => void;
+  showDomain: boolean;
+}
+
+function Breadcrumbs(props: Props): ReactElement {
+  const { path, onSelect, showDomain } = props;
+
+  assertAbsolutePath(path);
+  const { domain } = useContext(ProviderContext);
+
+  if (path === '/') {
+    return (
+      <h1 className={styles.breadCrumbs}>
+        <span className={styles.crumb} data-current>
+          {domain}
+        </span>
+      </h1>
+    );
+  }
+
+  // Remove leading /
+  const crumbs = path.slice(1).split('/');
+
+  return (
+    <h1 className={styles.breadCrumbs}>
+      {showDomain && (
+        <>
+          <button
+            className={styles.crumbButton}
+            type="button"
+            onClick={() => onSelect('/')}
+          >
+            <span className={styles.crumb}>{domain}</span>
+          </button>
+          <FiChevronRight className={styles.separator} title="/" />
+        </>
+      )}
```

Comment:
I found clearer to duplicate a bit of code here for the domain crumb rather than trying to manipulate the `crumbs` array

Reply:
Might as well create a `Crumb` component and move all three components into a `breadcrumbs-bar` folder, maybe?